certified-connectors/Narvar/readme.md

```diff
@@ -0,0 +1,20 @@
+# Title
+Narvar connector with Dynamics 365 Inteligent Order Management provides possibility to provide delivery information to end users.
+
+## Prerequisites
+First, contact your Narvar Customer Support Manager (or email: partnerships@narvar.com) to enable Microsoft Dynamics 365 Intelligent Order Management for your account.
+
+## Supported Operations
+The connector supports the following operations:
+* ```Post-CreateUpdate_Orders```: This will create a new order or send new information about an existing order
+* ```Get-Retrieve_Order```: This operation is used to retrieve order details
+* ```Put-CreateUpdate_Shipments```: This will update an order with a shipment record or change
+
+## Obtaining Credentials
```

Comment:
@BekaAre can you please provide clear steps? can you please provide a link?

Reply:
Can you provide more information? What kind of link is required?

Order is created by Calling Post method "PostCreateUpdateOrders" and passing data. That is enough for ideal case. If order exists at Narvar portal, it will notify end user about parcel delivery.
"GetRetrieveOrder" is needed to check order existance.
"PutCreateUpdateShipments" is needed to update only shipment information.

Is this information what you are asking?

Reply:
@BekaAre I am referring to the Obtaining Credentials section.